src/core/dfn-panel.runtime.js

```diff
@@ -32,30 +43,50 @@ function dfnPanel() {
       }
       case "hide": {
         hidePanel(panel);
+        panel = null;
         break;
       }
     }
-  });
+  };
+}
+
+/**
+ * @param {MouseEvent|KeyboardEvent} event
+ */
+function deriveCoordinates(event) {
+  if (event instanceof MouseEvent) {
+    return { x: event.clientX, y: event.clientY };
```

Comment:
I did notice the dialog being in the incorrect position when running with NVDA. I think this is because NVDA uses top left of the bounding box as clientX and clientY when clicking. Not sure what we can do about it (and whether it is an issue that needs resolving)

Reply:
Yeah, I noticed that Chrome + VO sometimes does the same thing... but sometimes it makes the event's `.clientX = 0`. I'd be hesitant to fix that unless it becomes a problem, as it seems to be a browser bug.

Reply:
I'm wondering if we get a zero (highly unlikely) we simulate a click in the middle of the bounding box

Reply:
Chrome indeed sends clientX/clientY zeros... this kinda bad because it makes AT detectable. Anyway, added some code to handle that.